department-of-veterans-affairs/vets-website
# Update special monthly pension page

## Summary
Update 'Special monthly pension' page on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the 'Special monthly pension' page in the 'Health and employment information' chapter
3. Verify that the deprecated fields have been updated
4. Verify that the field is required
5. Verify that the application state and data is saved to redux and submitted to the api
6. Verify that 'Special monthly pension' is able to be viewed and edited on the 'Review application' page
7. Verify that the application is able to be submitted

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77471)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After | Review |
| ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_form-saved (5)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/a1607e8a-6f98-427d-a648-36bb8d43c3df)|![localhost_3001_pension_application_527EZ_form-saved (4)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/b13f2ac2-eae1-4e90-9da8-b8bb35f2a9cc)|![Screenshot 2024-03-05 at 2 08 40 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/a8e385fb-e928-4c65-a093-43802844d2e0)




## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
